src/index.html

```diff
@@ -1,13 +1,305 @@
+                  src="img/image6.png"
+                  alt="#"
+                  class="details__img"
+                >
+              </div>
+            </div>
+            <h3 class="details__article-header">
+              Hydraulic disc brakes & Lightweight
+            </h3>
+            <p class="details__description">
+              Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </p>
+          </article>
+        </div>
+        <div class="details__link">
+          <a href="#" class="button">Explore</a>
+        </div>
+      </section>
+
+      <section class="contacts" id="contacts">
+        <h2 class="contacts__header">Contact us</h2>
+        <div class="contacts__wrapper">
+          <section class="contacts__input-section">
+            <form action="#" class="contacts__form">
+              <input
+              type="text"
+              class="contacts__name"
+              placeholder="Name"
+              >
+
+              <input
+              type="text"
+              class="contacts__email-input"
+              placeholder="Email"
+              >
+
+              <textarea
+              placeholder="Message"
+              name="coments"
+              class="contacts__message"
+              ></textarea>
+
+              <button type="submit" class="button contacts__button">Send</button>
+            </form>
+          </section>
+          <section class="contacts__info">
+            <a  href="tel:+123455555555"  class="contacts__link-containeer">
+              <span class="contacts__link-name">
+                Phone
+              </span>
+
+              <span class="contacts__link-content">
+                +1 234 5555-55-55
+              </span>
+            </a>
+
+            <a href="mailto:hello@miami.com" class="contacts__link-containeer">
+              <span class="contacts__link-name">
+                Email
+              </span>
+
+              <span class="contacts__link-content">
+                hello@miami.com
+              </span>
+            </a>
+
+            <a
+              href="https://www.google.com.ua/maps/place/400+N+1st+Ave+STE+700,+Minneapolis,+MN+55401,+%D0%A1%D0%BF%D0%BE%D0%BB%D1%83%D1%87%D0%B5%D0%BD%D1%96+%D0%A8%D1%82%D0%B0%D1%82%D0%B8+%D0%90%D0%BC%D0%B5%D1%80%D0%B8%D0%BA%D0%B8/@44.9813296,-93.275706,17z/data=!3m1!4b1!4m6!3m5!1s0x52b3328fdf233887:0x3348e5113ed2472d!8m2!3d44.9813258!4d-93.2731311!16s%2Fg%2F11nyn_tfyt?entry=ttu"
+              class="contacts__link-containeer"
+              target="_blank"
+             >
+              <span class="contacts__link-name">
+                Address
+              </span>
+
+              <span class="contacts__link-content">
+                400 first ave.<br>suite 700<br>Minneapolis, MN 55401
+              </span>
+            </a>
```

Comment:
for your phone, email and address add the same hover effect as you added to the buttons. (the more dynamic your landing page is, the better)